static/core/css/base.css

```diff
@@ -505,7 +505,9 @@ header .navbar-toggler-icon {
     height: 3em;
     line-height: 1.5;
     letter-spacing: normal;
```

Comment:
This class is also used on / and services, and making text bigger of / breaks the layout. I think we should make tree different classes. 

![image](https://github.com/Monadical-SAS/monadical.com/assets/94755911/ceaa32c7-cf65-4c6a-be83-7590809e63a9)